converter/src/main/java/gov/cms/qpp/conversion/xml/XmlUtils.java

```diff
@@ -1,20 +1,20 @@
 
 package gov.cms.qpp.conversion.xml;
 
-import java.io.UncheckedIOException;
-import java.nio.charset.Charset;
-import org.jdom2.Element;
-import org.jdom2.JDOMException;
-import org.jdom2.input.SAXBuilder;
-
 import java.io.ByteArrayInputStream;
 import java.io.IOException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.nio.charset.Charset'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)